bin/server.js

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
+import Debug from 'debug';
+import base from 'taskcluster-base';
+import api from '../lib/api';
+import path from 'path';
+import common from '../lib/common';
+import Promise from 'promise';
+import _ from 'lodash';
+
+let debug = Debug('secrets:server');
+
+/** Launch server */
+let launch = async function(profile) {
+  debug("Launching with profile: %s", profile);
+  var cfg = common.loadConfig(profile);
+
+  try {
+    var statsDrain = common.buildInfluxStatsDrain(
+      cfg.get('influx:connectionString'),
+      cfg.get('influx:maxDelay'),
+      cfg.get('influx:maxPendingPoints')
+    );
+  } catch(e) {
+    debug("Missing influx_connectionStraing: stats collection disabled.");
+    var statsDrain = common.stdoutStatsDrain;
+  }
+
+  // Start monitoring the process
+  base.stats.startProcessUsageReporting({
+    drain:      statsDrain,
+    component:  cfg.get('taskclusterSecrets:statsComponent'),
+    process:    'server'
+  });
+
+  let validator = await common.buildValidator(cfg);
+
+   let entity = base.Entity.configure({
+    version:          1,
+    signEntities:     cfg.get('azure:signEntities'),
+    partitionKey:     base.Entity.keys.StringKey('namespace'),
+    rowKey:           base.Entity.keys.StringKey('key'),
+    properties: {
+      namespace:      base.Entity.types.String,
+      key:            base.Entity.types.String,
+      value:          base.Entity.types.EncryptedJSON,
+      expires:        base.Entity.types.Date
+    }
+  }).setup({
```

Comment:
I would put the definition of this in a different file... something called data.js typically... Just to keep it simple, when/if we add additional `.configure()` with more versions... if we do migration... I doubt we'll need it here through.


Also the result from base.Entity.configure().setup() is actually a class. so I would call that `Secret` with capital S. and use `secret` as variable for instances of that class.